database/acid/services.js

```diff
@@ -0,0 +1,16 @@
+const { async } = require("regenerator-runtime");
```

Comment:
You don't need to use it this way. 
`const doSomething = async () => {
  console.log(await doSomethingAsync())
}`